src/components/ItemEditorPage/LeftPanel/Header/Header.container.ts

```diff
@@ -19,11 +19,13 @@ const mapState = (state: RootState): MapStateProps => {
     const collections = getCollections(state)
     collection = collections.find(collection => collection.id === collectionId)
   }
+  const address = getAddress(state)
   return {
-    address: getAddress(state),
+    address,
     isLoggedIn: isLoggedIn(state),
     isReviewing: isReviewing(state),
-    collection
+    collection,
+    hasEditRights: collection !== undefined && address != undefined && hasViewAndEditRights(state, address, collection)
```

Comment:
Hi @LautaroPetaccio, is there any reason why in the first condition we are using `!==` and in the second `!=`? Also which is the benefit of doing that and not directly checking existence?

`hasEditRights: collection && address && hasViewAndEditRights(state, address, collection)`

Reply:
Hi Juani! You're right, it was a typo, I forgot to add the other `=`. Fixed it!